test/unit/math/prim/mat/vectorize/foo_fun.hpp

```diff
@@ -10,13 +10,14 @@ namespace stan {
 
     /**
      * Example of how to define a functor for a vectorized function.
-     * The example includes a constrained version of exp().
+     * The example includes a constrained, punctured version of exp().
```

Comment:
I think so because I think it can be useful as a sandbox function for any changes to the unary vectorization framework or testing framework.